src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx

```diff
@@ -125,34 +125,49 @@ If you need to install the agent manually, follow these steps:
 
     1. [Install the New Relic native Android agent](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio).
 
-    2. Update `build.gradle`:
-       ```groovy
-         buildscript {
-           ...
-           repositories {
-             ...
-             mavenCentral()
-           }
-           dependencies {
-             ...
-             classpath "com.newrelic.agent.android:agent-gradle-plugin:6.11.1"
-           }
-         }
-       ```
-
-    3. Update `app/build.gradle`:
-       ```groovy
-         apply plugin: "com.android.application"
-         apply plugin: 'newrelic' // <-- add this
-       ```
-
-    4. Make sure your app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml`:
+    2. Add the following changes to apply the Gradle Pplugin:
+
+    If your project is using Plugins DSL (Flutter 3.16 or Later), add this snippet:
+
+      In `android/app/build.gradle`:
+        ```groovy
+        plugins {
+            id "com.android.application"
+            id "kotlin-android"
+            id "com.newrelic.agent.android" version "7.5.1"  //<-- include this
+        }
+        ```
+
+      Or, if your project is using the traditional way to apply the plugin, add this snippet:
```

Comment:
"traditional way" seems odd to me. But would this make sense to someone who regularly uses this plugin?